src/hungergames/tasks/LoadGamesTask.php

```diff
@@ -4,6 +4,8 @@
 use pocketmine\scheduler\Task;
 use pocketmine\utils\TextFormat;
 
+use pocketmine\Server;
```

Comment:
what's this supposed to fix?

Reply:
idk but i tried something, it was giving error to my console

Reply:
now, it doesn't give any error. but i think it didn't belong to it xd may it be? idk xd